projects/plugins/boost/app/assets/src/js/pages/settings/Settings.svelte

```diff
@@ -15,14 +17,25 @@
 	const shouldGetStarted = derived( [ config, connection ], ( [ $config, $connection ] ) => {
 		return $config.site.getStarted || ( ! $connection.connected && $config.site.online );
 	} );
+
+	$: activeModules = Object.entries( $modulesState ).reduce( ( acc, [ key, value ] ) => {
+		if ( key !== 'image_guide' && key !== 'image_size_analysis' ) {
+			acc.push( value.active );
+		}
+		return acc;
+	}, [] );
 </script>
 
 <ReRouter to="/getting-started" when={$shouldGetStarted}>
 	<div id="jb-dashboard" class="jb-dashboard jb-dashboard--main">
 		<Header />
 
 		<div class="jb-section jb-section--alt jb-section--scores">
-			<Score />
+			<Score
+				{activeModules}
+				criticalCssCreated={$criticalCssState.created}
+				criticalCssIsGenerating={$isGenerating}
```

Comment:
During the refactor, I prefer to pull these up to JS scope so that the code that touches component has no `$` store connection - I think that's going to make refactoring easier down the line when we need to actually refactor those reactive values.

Reply:
I've moved the stores to the Main component.

However, is there an alternative to passing props down the chain?

I kind of don't like doing it this way:
https://github.com/Automattic/jetpack/pull/33101/files#diff-cb1f7a8b9eab8e3b52de49d7a814a20ac5be9d6e2109ac5af2ec15c0e2d5277eR47-R49

and

https://github.com/Automattic/jetpack/pull/33101/files#diff-02098ca8bbf7a7f4d25f317f6e9360afa2a7d75284c249bbb53d0c72567cb7b6R30